src/main/java/de/tum/in/www1/artemis/domain/Course.java

```diff
@@ -561,4 +564,12 @@ public String getCourseArchivePath() {
     public void setCourseArchivePath(String courseArchiveUrl) {
         this.courseArchivePath = courseArchiveUrl;
     }
+
+    public int getMaxPoints() {
+        return maxPoints != null ? maxPoints : 0;
```

Comment:
I'm not sure if this implementation is good. In my opinion there could be a difference between max points being null (= not set) and max points being 0

Reply:
@Nullable